_posts/2017-07-28-reproducible-research-badges.md

```diff
@@ -0,0 +1,154 @@
+
+![scienceopen badges](/public/images/2017-07-28-badges/scienceOpen.png "Figure 1: ScienceOpen badges")
+<p class="attributionInlineImage">Figure 1: ScienceOpen badges</p>
+
+
+These are helpful when using the ScienceOpen website, but they’re not available for other websites. Additional issues are the four different styles and that some relevant information for reproducible geosciences is not available, e.g. executability status or the research location.
+
+Badges are also used directly for publications, leaving out the search engine “middleman”. Usually the published document, poster or presentation contains a badge along with the information needed to access the data or code. For example, the [Center for Open Science](https://cos.io/) [designed badges](https://osf.io/tvyxz/wiki/home/) for scientific articles. University of Washington’s [eScience Institute](http://escience.washington.edu/) created a peer-review process for open data and open materials badges (https://github.com/uwescience-open-badges/about) based on the COS badges. The service is meant for faculty members and students at the University of Washington, but even external researchers can apply. They also have a great list of [publications on the topic](https://github.com/uwescience-open-badges/about#where-can-i-read-more-about-this).
+
+A study [[1](#kidwell)] by Mallory C. Kidwell et. al. demonstrates that introducing such open data badges had a positive effect in the *Psychological Science* journal: After the journal started awarding badges for open data, more articles stated open data availability. The. They see badges as a simple yet effective way to promote data publishing. 
+
+Although these two examples are limited to a specific journal or institution respectively, they demonstrate the potential impact of badges. That is why our goal is to explore sophisticated and novel badge types (reproducibility, research location, etc.) and to find out how to provide them independently from a specific journal, conference, or website.
+
+## o2r-badger
+
+Advanced badges to answer the above questions come in handy when doing literature research as they open new ways of exploring research. But how can we get the required information for these badges? Some of these questions, such as the publication date, the peer review status and the open access status, can already be answered by online research library APIs, i.e. [Crossref](https://www.crossref.org/) or DOAJ[https://doaj.org/]. The [o2r API](http://o2r.info/o2r-web-api/) can (or will be able to) answer the remaining questions about reproducibility and location: Knowing if a publication is reproducible is a core part of this project. Furthermore, the location on which a research paper focuses, is extracted using o2r-meta (https://github.com/o2r-project/o2r-meta) by analyzing spatial files in the workspace of the publication. How can we integrate the information from these different sources?
```

Comment:
Strongly suggest to use a lot more new lines :-). It is fine to put each sentence on an extra line, makes commenting and revisioning much easier.

Anyway, make o2r-meta an actual link, please.